static/script.js

```diff
@@ -33,7 +33,7 @@ $(function () {
         newSessionStorageValue = ids.filter(item => item !== rowId)
       }
     }
-    newSessionStorageValue = newSessionStorageValue.filter(function(e){return e})
+    newSessionStorageValue = newSessionStorageValue.filter(function (e) { return e })
```

Comment:
This line is very cryptic. I assume this filter implicitly remove all empty records.
If you know that "empty" means empty string, you can use it to make it more explicit.

In line 33 you are using fat arrow function expression. Be consistent and use it everywhere when it fits best. One line fiter is a good place for it.

```javascript
newSessionStorageValue = newSessionStorageValue.filter(item => item !== '')
```

Reply:
good tip !